account_invoice_line_sale_line_position/readme/CREDITS.md

```diff
@@ -1,2 +1,5 @@
 The migration of this module from 13.0 to 14.0 was financially supported
 by Camptocamp
+
```

Comment:
```suggestion

```

Reply:
How about `https://github.com/OCA/account-invoice-reporting/pull/310`?
Do you think it's okay? I haven't seen you review it yet.